packages/in_app_purchase/in_app_purchase_storekit/example/shared/RunnerTests/Stubs.m

```diff
@@ -144,28 +144,6 @@ - (instancetype)initWithMap:(NSDictionary *)map {
 
 @end
 
-@interface InAppPurchasePluginStub ()
```

Comment:
This plugin stub needed to be converted to swift as you cannot subclass a swift class in objective C.